packages/debugger/src/executor.ts

```diff
@@ -50,9 +51,9 @@ export class CKBDebugger implements Executor {
    */
   private saveTmpTxFile(txSkeleton: TransactionSkeletonType): string {
     const debuggerData = parseDebuggerData(txSkeleton, this.loader);
-
-    // TODO replace with random tmp file name to avoid conflict
-    const tmpTxPath = path.join(os.tmpdir(), "ckb_debugger_tx.json");
+    const randomHash = crypto.randomBytes(18).toString('hex');    
+    const tempFileName = `lumos-debugger-data-${randomHash}`    
```

Comment:
```suggestion
    const randomHex = crypto.randomBytes(18).toString('hex');    
    const tempFileName = `lumos-debugger-data-${randomHex}`    
```

"randomHex" would be a more suitable term since it does not refer to a hash.